src/components/LayoutTree.tsx

```diff
@@ -518,10 +527,50 @@ const LayoutTree = () => {
 
       // set the width of table col1 to the minimum width of all visible thoughts in the column
       if (node.visibleChildrenKeys) {
-        const tableCol1Width = node.visibleChildrenKeys?.reduce(
+        let tableCol1Width = node.visibleChildrenKeys?.reduce(
           (accum, childKey) => Math.max(accum, sizes[childKey]?.width || 0),
           0,
         )
+
+        // Implementation of col1width = avg1/(avg1+avg2) * totalWidth
+        // with checks for minumum and maximum lengths.
+        if (node.childrenValues?.length) {
+          let col1Sum = 0
+          let col2Sum = 0
+
+          // Count how many entities are in the 2nd column.
+          // We don't do it for the 1st column, because
+          // we assume the row cannot exist without
+          // at least the 1st column's node.
+          let numCol2Entities = 0
+          let tableCol2Width = 0
+
+          node.childrenValues?.forEach(x => {
+            col1Sum += x.col1Value?.length || 0
+
+            if (x.col2Value?.length) {
+              col2Sum += x.col2Value?.length || 0
+              numCol2Entities++
```

Comment:
The calculation of the width of column 2 does not seem to account for any subthoughts in the column except for the first one of each row. I believe we should look at the length of the longest subthought in the column's row rather than only the first of each row.

In the video below, you can see that the length of the "Baz" subthought affects the layout, but the "Foo" subthought does not:

https://github.com/user-attachments/assets/ad735afa-52a8-4640-be32-93cc31e0c5e4